packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -168,10 +187,41 @@ mixin RenderConstrainedLayoutBuilder<ConstraintType extends Constraints, ChildTy
     markNeedsLayout();
   }
 
-  /// Invoke the layout callback.
-  void layoutAndBuildChild() {
+  bool _needsBuild = true;
+
+  /// Marks this layout builder as needing to rebuild.
+  ///
+  /// The layout build rebuilds automatically when layout constraints change.
+  /// However, we must also rebuild when the widget updates, e.g. after
+  /// `setState`, or `didChangeDependencies`, even when the layout constraints
+  /// remain unchanged.
+  ///
+  /// See also:
+  ///  * [ConstrainedLayoutBuilder.builder], which is called during the rebuild.
```

Comment:
you need a blank line before bulleted lists in our markdown

Reply:
Done.